src/array/geometrycollection/array.rs

```diff
@@ -125,7 +125,7 @@ impl<'a, O: OffsetSizeTrait> GeometryArrayTrait<'a> for GeometryCollectionArray<
     /// This operation is `O(1)` as it amounts to increase two ref counts.
     /// # Examples
     /// ```
-    /// use arrow2::array::PrimitiveArray;
+    /// use arrow::array::PrimitiveArray;
```

Comment:
I just finished a migration from arrow2 to arrow-rs and the docstrings haven't been updated yet (not that they were accurate before either; they were copied from arrow2). Could you revert these specific changes to make it clear that these docstring examples don't work?

Reply:
Sure.

Reply:
Thanks, there's a whole bunch of these docstrings that need to be updated, so ideally we'll be intentional about fixing them, ideally restoring `doctest = true` as well https://github.com/geoarrow/geoarrow-rs/blob/29930b0054c2ff7cb69650e7e9b754f822870106/Cargo.toml#L58-L60